packages/server/src/service/pdfForm/westVirginia.ts

```diff
@@ -0,0 +1,140 @@
+
+export const fillWestVirginia = (
+  stateInfo: WestVirginiaInfo
+) => fillFormWrapper(
+  'West_Virginia.pdf',
+  async ({check, text, placeImage}) => {
+
+    const { nameParts } = stateInfo
+
+    // First name.
+    text(nameParts.first, 0, 277, 117)
+    // Middle initial.
+    if (nameParts.middle) {
+      text(nameParts.middle, 0, 432, 117)
+    }
+    // Last name.
+    text(nameParts.last, 0, 132, 117)
+    // Suffix.
+    if (nameParts.suffix) {
+      text(nameParts.suffix, 0, 572, 117)
+    }
+
+    // Birth year.
+    const birthMonth = stateInfo.birthdate.split('/')[0]
+    const birthDay = stateInfo.birthdate.split('/')[1]
+    const birthYear = stateInfo.birthdate.split('/')[2]
+    text(birthMonth[0], 0, 480, 154)
+    text(birthMonth[1], 0, 490, 154)
+    text(birthDay[0], 0, 515, 154)
+    text(birthDay[1], 0, 525, 154)
+    text(birthYear[0], 0, 547, 154)
+    text(birthYear[1], 0, 557, 154)
+    text(birthYear[2], 0, 567, 154)
+    text(birthYear[3], 0, 577, 154)
+
+    // 'in the city/county of:' (locale)
+    text(stateInfo.county ? stateInfo.county : '', 0, 460, 137)
+
+
+    // Street and street number.
+    const streetAddress = stateInfo.address.streetNumber + ' ' + stateInfo.address.street + (stateInfo.address.unit ? ',' : '')
+    const streetAddressStart = 195
+    text(streetAddress, 0, streetAddressStart, 137)
+    // Apt/suite
+    const addressPosition = streetAddress.length * 7 + streetAddressStart
+    text(stateInfo.address.unit ? stateInfo.address.unit : '', 0, addressPosition, 137)
+    // City
+    text(stateInfo.address.city ? stateInfo.address.city : ' ', 0, 130, 154)
+    // Zip code.
+    text(stateInfo.address.postcode[0], 0, 367, 154)
+    text(stateInfo.address.postcode[1], 0, 377, 154)
+    text(stateInfo.address.postcode[2], 0, 387, 154)
+    text(stateInfo.address.postcode[3], 0, 397, 154)
+    text(stateInfo.address.postcode[4], 0, 407, 154)
+
+    // Mailing address.
+    // TODO: This s currently operating on the assumption that the mailing and regular address
+    // are the same. This is likely an unsafe assumption. mailingAddress should also be 
+    // an object of type address in stateInfo
+    const mailingAddressStart = 150
+    text(streetAddress, 0, mailingAddressStart, 175)
+    // Apt/suite
+    const mailingPosition = streetAddress.length * 7 + mailingAddressStart
+    text(stateInfo.address.unit ? stateInfo.address.unit : '', 0, mailingPosition, 175)
+    // City
+     text(stateInfo.address.city ? stateInfo.address.city : ' ', 0, 130, 192)
+    // state
+    text("WV", 0, 295, 192)
+    // Zip code.
+    text(stateInfo.address.postcode[0], 0, 367, 192)
+    text(stateInfo.address.postcode[1], 0, 377, 192)
+    text(stateInfo.address.postcode[2], 0, 387, 192)
+    text(stateInfo.address.postcode[3], 0, 397, 192)
+    text(stateInfo.address.postcode[4], 0, 407, 192)
+
+    // Phone number.
+    const phoneNumber = cleanPhoneNumber(stateInfo.phone)
```

Comment:
Same here on not splitting individual digits.

Reply:
Done.